packages/next/src/server/config-shared.ts

```diff
@@ -428,12 +428,14 @@ export interface ExperimentalConfig {
   /**
    * Enable filesystem cache for the turbopack dev server.
    *
-   * Defaults to `true` in canary releases.
+   * Defaults to `true`.
    */
   turbopackFileSystemCacheForDev?: boolean
```

Comment:
We need to:  
\- Add a non-experimental field to `config.turbopack`​. Bikeshed the final name and schema.  
\- Deprecate the field in the experimental schema.  
\- If there is a value set on the experimental field, copy the value to the non-experimental field.  
\- Update the docs to stop referring to the experimental field for `dev`​. Leave some documentation for the experimental flag on `build`​.  
  
This can be a follow-up.